src/Lecture2.hs

```diff
@@ -181,7 +237,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing []  = True
+isIncreasing [_] = True
+isIncreasing (x : y : rest)
+  | x > y     = False
+  | otherwise = isIncreasing (y : rest)
```

Comment:
You can the implementation slightly more elegant by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```